posthog/management/commands/start_temporal_worker.py

```diff
@@ -20,9 +20,41 @@ def add_arguments(self, parser):
         parser.add_argument(
             "--temporal_port", default=settings.TEMPORAL_SCHEDULER_PORT, help="Port for Temporal Scheduler"
         )
+        parser.add_argument("--namespace", default=settings.TEMPORAL_NAMESPACE, help="Namespace to connect to")
+        parser.add_argument("--task-queue", default=settings.TEMPORAL_TASK_QUEUE, help="Task queue to service")
+        parser.add_argument("--server-root-ca-cert", help="Optional path to root server CA cert")
+        parser.add_argument("--client-cert", help="Required path to client cert")
+        parser.add_argument("--client-key", help="Required path to client key")
```

Comment:
Since these are required, it would be better to set `required=True` and enforce that.

Reply:
Unless they are not always required, in which case I'd tweak up the `help` a bit to explain under which conditions.